modules/tinymce/src/plugins/image/main/ts/ui/MainTab.ts

```diff
@@ -71,6 +71,8 @@ const makeItems = (info: ImageDialogInfo) => {
     ]
   };
 
+  const dialogType = (hasColumns: boolean) => hasColumns ? { type: 'grid', columns: 2 } : { type: 'panel' };
```

Comment:
Should the parameter here match the value that's coming in?

```suggestion
  const dialogType = (hasImageCaption: boolean) => hasColumns ? { type: 'grid', columns: 2 } : { type: 'panel' };
```

Reply:
Maybe something like this instead, since it's purpose seems to be for general use?
```suggestion
  const getDialogContainerType = (useColumns: boolean) => useColumns ? { type: 'grid', columns: 2 } : { type: 'panel' };
```

Reply:
updated